pdr_backend/pdr_dashboard/test/test_util_db.py

```diff
@@ -98,7 +99,7 @@ def test_get_feed_daily_subscriptions_by_feed_id(_sample_app):
 
     result = db_mgr.feed_daily_subscriptions_by_feed_id(feed_id)
     all_subscriptions = db_mgr._query_db(
-        f"SELECT * FROM {Subscription.get_lake_table_name()}"
+        f"SELECT * FROM {tbl_parquet_path(db_mgr.lake_dir, Subscription)}"
```

Comment:
I think it's warranted to have a function in the the mapped tables that takes the lake dir parameter and returns the parquet path. Even with the parameter, we avoid mixture of concerns by importing stuff from the duckdb data store here.

Reply:
Well, if we send the lake dir as parameter, we still have to import from duckdb data store here to get the dir

Reply:
not really. That will be on the (already imported) table :)

Reply:
how? Table doesn't know about duckdb or ppss, whenever it's used inside it's method those are sent as params